test/mockito_test.dart

```diff
@@ -9,6 +9,14 @@ class RealClass {
   String methodWithNamedArgs(int x, {int y}) => "Real";
   String methodWithTwoNamedArgs(int x, {int y, int z}) => "Real";
   String methodWithObjArgs(RealClass x) => "Real";
+  // "SpecialArgs" here means type-parameterized args. But that makes for a long
+  // method name.
+  String methodWithSpecialArgs(
```

Comment:
I think I would rather describe "specialArgs" then call out it is a method in the name.

typeParameterizedFn maybe?